docusaurus.config.js

```diff
@@ -85,6 +85,7 @@ const config = {
       prism: {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
+        additionalLanguages: ["ruby", "php"],
```

Comment:
note: We need those extra values here in order to render correctly the snippets in those languages